parser/parser_test.go

```diff
@@ -0,0 +1,51 @@
+package parser
+
+import (
+	"go/build"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func mkdir(t *testing.T, dir string) {
```

Comment:
instead of this 3 funcs, you could simply defined this:

```go
func must(t *testing.T, err error) {
   if err != nil {
      t.Fatal(err)
   }
}
```

So you can then easily do:

```go
must(t, os.MkdirAll(dir, 0755))
```